rest_docs/rest-catalog-open-api.yaml

```diff
@@ -814,6 +1423,34 @@ components:
                   Server's do not need to implement this.
                 nullable: true
 
+    LoadTableResponse:
+      description: Table metadata result when loading a table
+      content:
+        application/json:
+          schema:
+            type: object
+            required:
+              - metadata
+            properties:
+              config:
+                type: object
+                additionalProperties:
+                  type: string
+              metadata:
+                $ref: '#/components/schemas/TableMetadata'
+
+    CommitTableResponse:
```

Comment:
Do we want to specify the transaction type someplace, e.g. simple, create_table, replace_table, etc.? Alternatively that could go in a token claim or something similar. For example, if there are two create table (without replace) transactions started at the same time, then the first one commits. When the second one commits, a catalog may want to prevent the replace from happening if it wasn't specified.

Reply:
Right now this is just focused on normal table updates. I think we need to layer on CREATE TABLE and transactional semantics in a separate PR. I think it will probably look like adding an option to the commit and the create, but I've not gotten it completely worked out yet.